subscriptions/providers/apple_in_app/__init__.py

```diff
@@ -196,13 +196,18 @@ def _get_or_create_payment(self,
     def _handle_single_receipt_info(self,
                                     user: User,
                                     receipt_info: AppleLatestReceiptInfo) -> Optional[SubscriptionPayment]:
+        logger.debug('Receipt handling started.')
```

Comment:
Is it useful? I'd add some receipt ID here, so that you know which receipt handling started. Same for ending.